posix-configs/rpi2/mainapp.config

```diff
@@ -3,11 +3,13 @@ param set SYS_AUTOSTART 4001
 param set MAV_BROADCAST 1
 sleep 1
 param set MAV_TYPE 2
-df_mpu9250_wrapper start
-df_hmc5883_wrapper start
+df_lsm9ds1_wrapper start -R 4
+#df_mpu9250_wrapper start -R 10
```

Comment:
Yer usually it occurs once on startup which is fine. Feel free to experiment, maybe I would have to do a power cycle. In general I would default to your LSM driver because the MPU9250 still has two issues creeping in every now and then:
https://github.com/PX4/DriverFramework/issues/74
https://github.com/PX4/DriverFramework/issues/89